oem/firstvoices/android/app/src/main/java/com/firstvoices/keyboards/MainActivity.java

```diff
@@ -49,17 +49,17 @@ protected void onCreate(Bundle savedInstanceState) {
 
         setContentView(R.layout.activity_main);
 
+        if (BuildConfig.DEBUG) {
+          KMManager.setDebugMode(true);
+        }
+        KMManager.initialize(getApplicationContext(), KMManager.KeyboardType.KEYBOARD_TYPE_INAPP);
+
         FVShared.getInstance().initialize(this);
 
         FVShared.getInstance().upgradeTo12();
         FVShared.getInstance().upgradeTo14();
         FVShared.getInstance().preloadPackages();
 
-        if (BuildConfig.DEBUG) {
-            KMManager.setDebugMode(true);
-        }
-        KMManager.initialize(getApplicationContext(), KMManager.KeyboardType.KEYBOARD_TYPE_INAPP);
```

Comment:
Will this change in initialization order cause problems for upgrade scenarios?

Reply:
Unsure - atm I don't have a clear way to test upgrade scenarios.

Reply:
Previously, `KMManager.initialize()` hasn't happened yet, so the fv_all.kmp file hasn't been extracted.
This is why we've been copying `keyboards.csv` into the assets folder during the build.sh process (at the same folder as fv_all.kmp).